manifests/upstream/keptn/base/kustomization.yaml

```diff
@@ -48,4 +31,22 @@ images:
   - name: connecteverything/nats-server-config-reloader
     newName: docker.io/connecteverything/nats-server-config-reloader
   - name: synadia/prometheus-nats-exporter
-    newName: docker.io/synadia/prometheus-nats-exporter
\ No newline at end of file
+    newName: docker.io/synadia/prometheus-nats-exporter
```

Comment:
I think these names need to match without docker.io and the newName with it

Reply:
I don't quite follow what you said?

Reply:
All deployments/statefulsets' container image in `manifests/keptn.yaml` is starting with `docker.io` and I have tested them in the deploy phase.

Reply:
I think I am getting confused with the generated_templates - Maybe we should exclude all the generated templates from Git and only commit the resulting manifest ?

Reply:
We can git-ignore the `upstream/*/helm/generated_templates` but when we make a change to kustomization layer e.g. `upstream/keptn/base`, we will also need to make a change to `upstream/keptn/helm` so that the `make` command will rebuild the `generated_templates` so kustomize can read these files build the `keptn.yaml`. That is not very intuitive.

If all files under `manifests` are packed into karina binary. I would suggest moving `upstream` out of the `manifests` directory so that any files in the `upstream` won't be added to karina binary and we can still keep the upstream manifests in the git (there are benefits doing that - like when they changed the upstream's remote and we're no longer able to pull it).

What do you think?

Reply:
It isn't the binary bloat I am concerned about (the go dependencies are 100mb+) 

Also less concerned with the remote changing underneath as we are committing the result, linting should warn us in the event of a divergence where the most likely action would be to bump the version. 

Can we not make helm as a dependency before make kustomize ?

From a build timing perspective, it can take a little longer to build the initial chart as it will be running as a separate lint github action with the main linter probably taking much longer.